packages/components/pie-modal/vite.config.js

```diff
@@ -1,3 +1,20 @@
 import viteConfig from '@justeattakeaway/pie-components-config/vite.config';
 
-export default viteConfig;
+export default viteConfig({
+    build: {
+        lib: {
+            entry: {
+                index: 'src/index.ts',
+                react: 'src/react.ts',
+            },
+            formats: ['es'],
+        },
+        rollupOptions: {
+            external: [
+                'react',
+                /^lit/,
+                /@justeattakeaway/
```

Comment:
I'm guessing it'd just make sense to add this into the shared vite config potentially as a follow up (as all components should exclude these?)

Reply:
Yes, that is the logical progression but I didn't want to do too much at once in this PR